components/src/core/InfoListItem/InfoListItem.styles.tsx

```diff
@@ -0,0 +1,78 @@
+import { createStyles, makeStyles, Theme } from '@material-ui/core/styles';
+import { InfoListItemProps } from './InfoListItem';
+import color from 'color';
+import * as Colors from '@pxblue/colors';
+
+const getIconColor = (props: InfoListItemProps): string => {
+    const { avatar, iconColor, statusColor } = props;
+    if (iconColor) return iconColor;
+    if (avatar) {
+        return statusColor ? (color(statusColor).isDark() ? Colors.white[50] : Colors.black[500]) : Colors.white[50]; // default avatar is dark gray -> white text
+    }
+    return statusColor ? statusColor : 'inherit';
+};
+
+export const useStyles = makeStyles<Theme, InfoListItemProps>((theme: Theme) =>
+    createStyles({
+        root: {
+            backgroundColor: (props): string => props.backgroundColor || 'transparent',
+            cursor: (props): string => (props.onClick ? 'pointer' : 'default'),
+            height: (props): number => (props.dense ? 52 : 72),
+            '&:hover': {
+                backgroundColor: (props): string => (props.onClick ? 'rgba(0, 0, 0, 0.08)' : ''),
```

Comment:
how's this work for different themes?

Reply:
This uses the theme that is passed into the MUIThemeProvider.  I tested this out in storybook by setting the InfoListItem's root bg color to use the theme primary, adjusting the light and dark themes to have a difference (light = red, dark = green), then toggling between the themes in the config.js.

Reply:
Would it make more sense to have a separate element that goes from transparent to this semi-transparent black color over the existing background?

E.g., assume the user has the background color set to yellow (on a dark gray page). Right now the background would change from `yellow -> almost black` versus `yellow -> dark yellow`.

Reply:
This was a good catch, I didn't test the hover state on a deep color.  I added a separate element that handles the hover event.